File: 백준/Bronze/10430. 나머지/나머지.js
const fs = require('fs');

const line = fs.readFileSync(0, 'utf8').trim();
const [a, b, c] = line.split(' ').map(Number);

let answer = "";
answer += `${(a + b) % c}\n`;
answer += `${((a % c) + (b % c)) % c}\n`;
answer += `${(a * b) % c}\n`;
answer += `${((a % c) * (b % c)) % c}`;

console.log(answer);

File: 백준/Bronze/10430. 나머지/나머지.py
import sys

line = sys.stdin.readline
a, b, c = map(int, line().split())

answer = ""
answer += str((a + b) % c) + "\n"
answer += str(((a % c) + (b % c)) % c) + "\n"
answer += str((a * b) % c) + "\n"
answer += str(((a % c) * (b % c)) % c)

print(answer)